maligned.go

```diff
@@ -22,10 +23,19 @@ var fset = token.NewFileSet()
 func main() {
 	flag.Parse()
 
+	importPathPatterns := flag.Args()
+	if len(importPathPatterns) == 0 {
+		importPathPatterns = []string{"."}
```

Comment:
You're right, it does:

```bash
$ goexec 'gotool.ImportPaths([]string{"."})'
([]string)([]string{
	(string)("."),
})
$ goexec 'gotool.ImportPaths(nil)'
([]string)([]string{
	(string)("."),
})
```

Thanks for the tip, I'll remove it.

I'm not sure why I thought I needed it, I guess I missed it. The documentation doesn't specify this.